Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift

```diff
@@ -152,6 +155,7 @@ import BraintreeCore
             checkoutParameters["postal_code"] = shippingAddressOverride?.postalCode
             checkoutParameters["country_code"] = shippingAddressOverride?.countryCodeAlpha2
             checkoutParameters["recipient_name"] = shippingAddressOverride?.recipientName
+            checkoutParameters["recipient_email"] = userAuthenticationEmail
```

Comment:
For sure not in scope from our side but it's odd to me that on the gateway side we call this `payer_email` and for checkout we call it `recipient_email`

Reply:
It should be `payer_email`, updated!